File: pgwire-jdbc-server/pgwire-jdbc-bridge/src/main/java/org/kendar/pgwire/utils/SqlParseResult.java
package org.kendar.pgwire.utils;

public class SqlParseResult {
    private String value;
    private SqlStringType type;

    public SqlParseResult(String value, SqlStringType type)
    {
        this.value = value;
        this.type = type;
    }

    public String getValue() {
        return value;
    }

    public SqlStringType getType() {
        return type;
    }
}

File: ado/PgWireCoreClient/PgWireAdo/utils/SqlParseResult.cs
namespace PgWireAdo.utils;

public class SqlParseResult
{
    public SqlParseResult(string value, SqlStringType type)
    {
        Value = value;
        Type = type;
    }

    public string Value { get; set; }
    public SqlStringType Type { get; set; }
}